grails-app/views/profile/myprofile.gsp

```diff
@@ -39,10 +39,15 @@
                 </a>
             </li>
             <li>
-                <a href="${grailsApplication.config.biocache.search.url}?q=*%3A*&fq=assertion_user_id%3A%22${user.id}%22">
+                <a href="${grailsApplication.config.biocache.search.url}%22${user.id}%22">
```

Comment:
This URL has a double-slash in it in the current version based on ``ala-install`` https://github.com/AtlasOfLivingAustralia/ala-install/commit/dd82ffafa026d14b7f18e596aa6dcb84a3a8a18f

```
https://biocache.ala.org.au//occurrences/search/?q=*%3A*&fq=assertion_user_id%3A%2227583%22
```

instead of:

```
https://biocache.ala.org.au/occurrences/search/?q=*%3A*&fq=assertion_user_id%3A%2227583%22
```

There is the same issue for:

```
https://biocache.ala.org.au//download/myDownloads/
```

which should be

```
https://biocache.ala.org.au/download/myDownloads/
```

The auth ansible files in ansible-inventories are the exception in terms of having a trailing slash attached to the "base_url" properties, so it would be best to fix both the biocache and the other base_url properties there and redeploy.

Reply:
Anything that is a "base URL" should end in a /, I know everything else is missing it but everything else is wrong.

In this case, I would use a url builder to construct the search URL and pass it as a model param to the template.  Or create a URL builder link tag?

Spring provides a `UriComponentsBuilder` that should do the trick.

Reply:
I wouldn't mind if the whole ALA switched to putting trailing slashes on all base URLs if it was planned and implemented as a bulk effort. 

Just trying to keep the advantage of the current convention being the way it is everywhere rather than having it different across different inventories and not being sure which app supports which convention.